crates/sparrow-compiler/tests/snapshots/compiler_golden_tests__lookup_same_grouping.snap

```diff
@@ -217,8 +217,11 @@ operations:
         result_type:
           kind:
             List:
-              kind:
-                Primitive: 10
+              name: item
```

Comment:
Ugh. All these protos are a pain. This could be another thing we simplify if we just used serde for plans or something.